### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Dashboard_Databases_Screenreader : Table header is not announced by NVDA.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3160]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**-A11y_AzureDataStudioStayHealthy;-A11ySev2;-A11yTCS;-A11yMAS;-DesktopApp;-MAS1.3.1;-SQL Azure Data Studio;-FTP;-NVDA;-STP;-JAWS;-MAS2.4.6;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider (system setup)
Commit: 8288360
Date: 2020-05-29T05:54:49.731Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.3.1, MAS2.4.6
Screenreader : NVDA 2019.3.1,JAWS

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Double click on connected server or right click on it & select manage option to open the Dashboard.
4. Navigate to Databases under Dashboard & hit enter.
5.Navigate to databases table.
6.Start Screenreader as NVDA and observe whether NVDA is reading table headers or not. 

**Actual:**
NVDA is not reading the Databases table column header when we are navigating using up/down arrow key.

Similar issue also observed with JAWS.

**Expected:**
NVDA should read the Databases table column header when we are navigating using up/down arrow key.

**User Impact:**
Screen reader users will not get information about the table column header.
